products/views.py

```diff
@@ -1,3 +1,67 @@
-from django.shortcuts import render
+import json
+
+from django.http        import JsonResponse
+from django.views       import View
+
+from products.models    import Category, Product, ThemeProduct, Theme, Material, ProductImage, ProductOption, Size, Color
+from users.models       import Like
+
+class DetailView(View):
```

Comment:
보다 명확한 네이밍이 필요해보입니다!
```suggestion
class ProductDetailView(View):
```

Reply:
네이밍 변경하였습니다~!